test/spec/common/util/validationUtil.spec.js

```diff
@@ -0,0 +1,61 @@
+Bahmni.Registration.customValidator = {
+  "age.days": {},
+  "name": {},
+  "Telephone Number": {},
+  "date": {}
+};
+
+describe('ValidationUtil', function () {
+  var date = new Date();
+
+  var ValidationUtil = Bahmni.Common.Util.ValidationUtil;
+  var customValidator = Bahmni.Registration.customValidator;
+  var complexObject, objectConfiguration;
+  beforeEach(function () {
+    complexObject = {
+      "name": "valerio",
```

Comment:
Lets avoid using names from people we know. We better use random names: Foo Bar or movie characters: Freddy Krueger, Michael Myers, etc